### DISCUSSION TITLE
Migrate Self Hosted to Hosted

### DISCUSSION THREAD

**Discussion by ScarVite** [supabase/supabase#10751] [Questions] [answered]
Is there any way to migrate a self hosted instance into a hosted instance?

I have database access available, which means i can export my database into a pgsql dump.

**Comment by GaryAustin1** [comment, marked as answer]
Maybe… https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects#migrate-your-project

**Reply by ScarVite** [reply]
Must have overlooked this.

thanks :D
